src/main/java/com/hyperwallet/clientsdk/util/HyperwalletApiClient.java

```diff
@@ -132,6 +138,15 @@ protected void checkErrorResponse(final Response response) throws ParseException
         }
     }
 
+    private void checkResponseHeader(Response response) {
+        String contentTypeHeader = response.getHeader(CONTENT_TYPE_HEADER);
+        boolean isNotValidContentType = !contentTypeHeader.equals(VALID_JSON_CONTENT_TYPE) &&
+                !contentTypeHeader.equals(VALID_JSON_JOSE_CONTENT_TYPE);
```

Comment:
would this code be easier to read if we instead do the ff: `boolean isNotValidContentType = !(contentTypeHeader.equals(VALID_JSON_CONTENT_TYPE) || contentTypeHeader.equals(VALID_JSON_JOSE_CONTENT_TYPE))`

Reply:
Refactored this condition